style.css

```diff
@@ -1,41 +1,113 @@
 * {
-    padding: 0;
-    margin: 0;
+  padding: 0;
+  margin: 0;
+  /* outline: solid red 1px; */
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+}
+
+body {
+  background-color: white;
 }
 
 .mainWrapper {
-    height: 100vh;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
 .windowWrapper {
-    width: 300px;
-    height: 400px;
-    border-radius: 6px;
-    overflow: hidden;
-    box-shadow: 0 5px 10px rgba(154, 160, 185, 0.05),
-    0 15px 40px rgba(166, 173, 201, 0.2);
+  max-width: 320px;
+  border-radius: 10px;
+  overflow: hidden;
+  box-shadow: 0px 36px 100px 0px rgba(0, 0, 0, .2),
+    0px 0px 1px 0px rgba(0, 0, 0, .55)
 }
 
+.topWrapper {
+  height: 100px;
+  padding: 0px 10px;
+  background-color: black;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  align-content: flex-end;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: flex-end;
+}
 
-.inputWindow {
-    height: 80px;
-    width: 200px;
-    background-color: #f5f5f5;
-    border: none;
+.inputNumber {
+  /* font-size: 1.7rem; */
+  font-size: 2rem;
+}
+
+.previewCalc {
+  opacity: 50%;
 }
 
 .numpadWrapper {
-    max-width: 600px;
-    height: 200px;
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+}
+
+.topNumpad {
+  display: flex;
+}
+
+.bottomNumpad {
+  display: flex;
+}
+
+.operators {
+  display: flex;
+  flex-direction: column;
+}
+
+.numbers {
+  width: 240px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  direction: rtl;
 }
 
-.placeholder {
-    height: 80px;
-    width: 80px;
-}
\ No newline at end of file
+.calcButton {
+  height: 80px;
+  width: 80px;
+  border: none;
+  background-color: white;
+  font-weight: 400;
+  font-size: 1.3rem;
+  transition: all 0.06s;
+}
+
+.calcButton:hover {
+  transition: all 0.1s;
+  background: hsl(0, 0%, 90%) !important;
```

Comment:
why do you use `!important` here?
This is kind of hacky and should not be used.

Reply:
Changed it in the most recent commit. Thanks for the suggestion!